README.md

```diff
@@ -2,38 +2,67 @@ easy-manage-curation-work
 ===========
 [![Build Status](https://travis-ci.org/DANS-KNAW/easy-manage-curation-work.png?branch=master)](https://travis-ci.org/DANS-KNAW/easy-manage-curation-work)
 
-<!-- Remove this comment and extend the descriptions below -->
-
 
 SYNOPSIS
 --------
 
-    easy-manage-curation-work (synopsis of command line parameters)
-    easy-manage-curation-work (... possibly multiple lines for subcommands)
-
-
+    easy-manage-curation-work list [<easy-datamanager>]
+    easy-manage-curation-work assign <easy-datamanager> <UUID>
+    easy-manage-curation-work unassign [<easy-datamanager> [<UUID>]]
```

Comment:
I'm still not completely sold on this idea of making the `<UUID>` optional within the optional `<easy-datamanager>`. What happens when you only give a `<UUID>`? Then the application will recognize this as the `<easy-datamanager>` parameter, right?
I think my main problem is that the input arguments cannot be validated properly by Scallop, so the error propagates to the application (where it is ultimately detected as a 'bad datamanager id').
Any thoughts on how we could fix this? Perhaps use `opt` instead of `trailArg`?
@janvanmansum @lindareijnhoudt any suggestions?

Reply:
We (@lindareijnhoudt  and I) decided that we leave it as it is, anyway for now.